examples-gallery/plot_parameter_estimation.py

```diff
@@ -0,0 +1,205 @@
+# %%
+"""
+
+Parameter Estimation
```

Comment:
The title and filename are too generic.

1. We use the terms "parameter identification" in opty not "parameter estimation", we should be consistent.
2. This is a special parameter id example, in that it is showing how to use multiple measurement trials that constitute a set of discontinuous data. So the title should reflect this uniqueness (relative to the other simpler parameter id examples)